types/node/v8/index.d.ts

```diff
@@ -5575,6 +5576,48 @@ declare module "util" {
     export namespace promisify {
         const custom: symbol;
     }
+
+    export interface TextDecodeOptions {
+        fatal?: boolean;
```

Comment:
According to https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_textdecoder_decode_input_options the options for `decode()` is an object holding an optional property `stream`.